### ISSUE TITLE
Valkyrie doesn't find your scenario description

### ISSUE THREAD

**Issue by BenRQ** [Samurai83ITA/Scenari-Valkyrie#2]
Hi Samurai83ITA,

You have renamed your scenarios files without the accent of the original scenario.

There is a discrepancy between the folder name and the scenario name, and the Valkyrie tool to check for upgrade cannot find your files anymore.
Also players won't be able to download your scenario anymore.

Scenario is : 
UnaTristeVerità0 (with a 'à')

Files are : 
UnaTristeVerita0.ini 
UnaTristeVerita0.valkyrie 
(without a 'à')

Can you please bring back the 'à' ?

Thanks :)

**Comment by BenRQ** [comment]
@scrubbless  for info

**Comment by scrubbless** [comment]
I'd recommend we remove all accented characters and republish.

@vidimus you may be able to assist as I believe you provided the updated scenario with translations?

EDIT: I think I fixed this yesterday actually.

**Comment by vidimus** [comment]
> I'd recommend we remove all accented characters and republish.
> 
> @vidimus you may be able to assist as I believe you provided the updated scenario with translations?
> 
> EDIT: I think I fixed this yesterday actually.

Yes, for sure. I will create again the packages without accented characters on the title. Then I will send them to @Samurai83ITA

**Comment by vidimus** [comment]
One question, @scrubbless , changing the scenario name is not going to delete the plays stats? Because in that case, maybe it's better to bring the "à" back.

**Comment by BenRQ** [comment]
I can replace the name in the original stats so you don't lose it if required

**Comment by scrubbless** [comment]
I've replaced these out, but no worries as Ben has stated we can just find and replace the accented records with the non accented ones.

Either of us can do this in the "Valkyrie Live data" sheet so all connected sheets (for example Anton's mega sheet) will get updated too.

Incidentally this is how I censor the inappropriate comments. Just for the record I only change (and comment) this sheet if absolutely necessary, o don't like to meddle with the stats data.

**Comment by scrubbless** [comment]
> One question, @scrubbless , changing the scenario name is not going to delete the plays stats? Because in that case, maybe it's better to bring the "à" back.

No need to do this @vidimus I've got full access to the repo so I have taken the packages and repackaged them with non accented names.

**Comment by scrubbless** [comment]
Hi @Samurai83ITA 

This has happened again with your latested update, the scenario folders and .valkyrie files have unrecognisable characters again which is breaking the import of the scenario explorer:

![image](https://user-images.githubusercontent.com/32058089/145976016-0bfe47d9-d1e3-4cc5-8b8f-d78b7f41718c.png)


I am going to correct this again, but can you please rename the folders in your %appdata%\valkyrie\MoM\Editor folder and remove the accented 'a' from each of your scenario folders before publishing any updates please.
